modules/apps/fragment/fragment-service/src/main/java/com/liferay/fragment/internal/upgrade/FragmentServiceUpgrade.java

```diff
@@ -106,9 +108,15 @@ public void register(Registry registry) {
 		registry.register(
 			"2.4.0", "2.5.0",
 			new com.liferay.fragment.internal.upgrade.v2_5_0.
-				UpgradeFragmentEntryLink());
+				UpgradeFragmentEntry(),
+			new com.liferay.fragment.internal.upgrade.v2_5_0.
+				UpgradeFragmentEntryLink(),
+			new UpgradeFragmentEntryVersion(_counterLocalService));
```

Comment:
Better to register a new "2.6.0" upgrade then reuse a step.  Hard to control the release process for a ticket, if it's separate it doesn't matter if the release has happened.